internal/distributed/proxy/httpserver/constant.go

```diff
@@ -94,14 +94,22 @@ const (
 
 	HTTPReturnHas = "has"
 
-	HTTPReturnFieldName         = "name"
-	HTTPReturnFieldID           = "id"
-	HTTPReturnFieldType         = "type"
-	HTTPReturnFieldPrimaryKey   = "primaryKey"
-	HTTPReturnFieldPartitionKey = "partitionKey"
-	HTTPReturnFieldAutoID       = "autoId"
-	HTTPReturnFieldElementType  = "elementType"
-	HTTPReturnDescription       = "description"
+	HTTPReturnFieldName             = "name"
+	HTTPReturnFieldID               = "id"
+	HTTPReturnFieldType             = "type"
+	HTTPReturnFieldPrimaryKey       = "primaryKey"
+	HTTPReturnFieldPartitionKey     = "partitionKey"
+	HTTPReturnFieldAutoID           = "autoId"
+	HTTPReturnFieldElementType      = "elementType"
+	HTTPReturnDescription           = "description"
+	HTTPReturnFieldIsFunctionOutput = "is_function_output"
```

Comment:
It seems more appropriate to use isFunctionOutput to keep it consistent with other forms.